src/utils/inputValidation/preloadedData.js

```diff
@@ -0,0 +1,56 @@
+import { isObject, isString } from '../lang';
+import logFactory from '../logger';
+const log = logFactory('', {
+  displayAllErrors: true
+});
+
+function validateSinceData(maybeSince, method) {
+  if (maybeSince > -1) return true;
```

Comment:
Be careful. If you do `maybeSince > -1` with undefined it yields false, but if you do it with null it yields true. The validation should check that's a number before anything, then you see how big it is. 

Null is probably what you'll get if the BE serializer can't assign a proper one or there's some sort of parsing error somewhere (remember the data serializer won't be running in this same app), given that most languages won't send undefined.